src/App.tsx

```diff
@@ -1,9 +1,94 @@
+import Navbar, { NavbarItem } from "./components/Navbar";
+import {
+  faClock,
+  faCompass,
+  faFutbol,
+  faUser,
+} from "@fortawesome/free-solid-svg-icons";
 import "./app.scss";
+import { useState } from "react";
+
+const initialItems = [
+  {
+    icon: faClock,
+    title: "Time",
+  },
+  {
+    icon: faFutbol,
+    title: "Comments",
+  },
+  {
+    icon: faCompass,
+    title: "Compass",
+  },
+  {
+    icon: faUser,
+    title: "User",
+  },
+] as const;
+
+type Tab = (typeof initialItems)[number]["title"];
+
+// bonus: make it headless
 
 function App() {
+  const [items, setItems] = useState([...initialItems]);
+  const [activeTab, setActiveTab] = useState<Tab>(initialItems[0].title);
+
+  const handleItemClicked = ({ title, id }: { title: Tab; id?: number }) => {
+    console.log(`Item clicked title ${title} id ${id}`);
+    setActiveTab(title);
+  };
+
+  const getContentTab = (activeTab: Tab) => {
```

Comment:
suggestion: We talked about this in a call last week. How can you improve this with O1 instead of On? (a direct lookup, remember?)

Also, as this variable does not depend on anything from React, it could be declared outside the component so it doesn't get redefined on every re-render.

Reply:
hehe i didn't pay much attention to it since it was an example but fair enough, even examples are supposed to be good quality! 😆

Reply:
In general, examples/docs must be the best quality, you know why? It's where most people copy from to their own code! :p  You don't want copilot to learn from your poor-quality examples ahah